lib/rex/logging/sinks/flatfile.rb

```diff
@@ -9,48 +9,16 @@ module Sinks
 # file on disk.
 #
 ###
-class Flatfile
-
-  include Rex::Logging::LogSink
+class Flatfile < Rex::Logging::Sinks::Stream
 
   #
   # Creates a flatfile log sink instance that will be configured to log to
   # the supplied file path.
   #
   def initialize(file)
-    self.fd = File.new(file, "a")
-  end
-
-  def cleanup # :nodoc:
```

Comment:
Should there still be a `cleanup` method to ensure the file is consistent if a file logger is replaced?  I do not see one in `Rex::Logging::Sinks::Stream`.